instance_selector/edit_handlers.py

```diff
@@ -1,16 +1,29 @@
 from django.template.loader import render_to_string
 from django.utils.safestring import mark_safe
-from wagtail.admin.edit_handlers import BaseChooserPanel
+from wagtail import VERSION as WAGTAIL_VERSION
 from instance_selector.widgets import InstanceSelectorWidget
 
+if WAGTAIL_VERSION >= (3, 0):
+    from wagtail.admin.panels import BaseChooserPanel
+else:
+    from wagtail.admin.edit_handlers import BaseChooserPanel
+
 
 class InstanceSelectorPanel(BaseChooserPanel):
```

Comment:
I was looking at: [Upgrade docs](https://docs.wagtail.org/en/stable/releases/3.0.html#api-changes-to-panels-edithandlers)

There's an opportunity to now use only FieldPanel so it's inline with the new Wagtail panels. I don't think this is a requirement but could be changed. But I guess the changes wouldn't apply to v<3.0 so it would need to be left in place there.

Reply:
I see your point here, and I do agree let's just keep it this way because it's not just for Wagtail 3